trace/tests/unit/test_trace.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import mock
+
+
+class TestTrace(unittest.TestCase):
+
+    project = 'PROJECT'
+
+    @staticmethod
+    def _get_target_class():
+        from opencensus.trace.trace import Trace
```

Comment:
I kind of want to completely abandon the Veneer style of importing things inside test methods and having all these _make_one() methods.  What do you think?